model/change_point_detection.go

```diff
@@ -328,6 +334,75 @@ func ReplaceChangePoints(ctx context.Context, env cedar.Environment, performance
 	return catcher.Resolve()
 }
 
+func GetTotalPagesForChangePointsGroupedByVersion(ctx context.Context, env cedar.Environment, projectId string, pageSize int) (int, error) {
+	countKey := "count"
+	pipe := appendAfterBaseGetChangePointsByVersionAgg(projectId, bson.M{
```

Comment:
Why do we need a separate function to append? Seems like we could just concatenate the stages it produces with other stages in any order we wish?

Reply:
These are the starting stages for both the totalPages aggregation and the getting the changepoints aggregation, I don't think there's a case, at least right now, where it makes sense to put these stages in any other order. As well, this change was originally requested by Sam [here](https://github.com/evergreen-ci/cedar/pull/200#discussion_r407820378).
 
If this is something you feel strongly about, please discuss with him so that you're not at cross purposes.

Reply:
I mean, I don't think that appendAfterSpecificSituation function is very useful anywhere else, so it seems like cleaning it up to be something like createStandardPipelineStage would make it more reusable since you can do whatever you want with that stage. I don't want to hold up this PR too much though, so go ahead like this.